javascripts/main.js

```diff
@@ -1,4 +1,5 @@
 // @ts-nocheck
+/* eslint prefer-arrow-callback: [ "error" ] */
```

Comment:
I think if you pull in this change, run `npm run prettier-fix` and then commit this should make CI pass:

```suggestion
/* eslint prefer-arrow-callback: [ "error" ] */
/* eslint arrow-parens: [ "error", "as-needed" ] */
```